tests/system/action/motion_workflow/test_create.py

```diff
@@ -16,6 +16,7 @@ def test_create(self) -> None:
         self.assert_status_code(response, 200)
         workflow = self.get_model("motion_workflow/1")
         assert workflow.get("name") == "test_Xcdfgee"
+        assert workflow.get("first_state_id") == 1
         state = self.get_model("motion_state/1")
         assert state.get("workflow_id") == 1
```

Comment:
Please check here that `first_state_of_workflow_id` is set correctly

Reply:
Add this assert, too. Done.